components/tools/OmeroWeb/omeroweb/webadmin/templates/webadmin/experimenter_form.html

```diff
@@ -398,7 +398,7 @@
     <div id="privilege_info"></div>
     <div id="privilege_list">
         <h1>Choose user privileges</h1>
-        <p>For more info see <a href="https://www.openmicroscopy.org/latest/omero/sysadmins/admins-with-restricted-privileges.html">administrator privileges</a>.</p>
+        <p>For more info see <a href="https://www.openmicroscopy.org/latest/omero/sysadmins/restricted-admins.html">administrator privileges</a>.</p>
```

Comment:
Just reviewed. Shouldn't it be `docs.openmicroscopy.org`?

Reply:
So it should!!